sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 1.5.2 (Unreleased)
+## 1.5.2 (2021-09-01)
 
+- Fixed a bug introduced on 1.5.1 that caused the `ManagedIdentityCredential` to fail authenticating in Arc environments.
```

Comment:
This can use some more details, can you use the details from the PR description here so that we have some context? Also, I assumed that the problem was introduced when we moved to core-rest-pipeline and that move happened in 1.5.0. So, why are we saying 1.5.1 here?

Reply:
You’re correct! I think I meant to say 1.5.0 and wrote 1.5.1 by mistake. I’ll fix this

Reply:
Updated! Review appreciated 🙏